src/main/java/com/epam/training/designpatterns/fastfoodrestaurant/food/Mayonnaise.java

```diff
@@ -0,0 +1,23 @@
+package com.epam.training.designpatterns.fastfoodrestaurant.food;
+
+public class Mayonnaise extends Condiment implements Food {
```

Comment:
Ha, ha, itt majonéz is van? :-)

Reply:
Igen, megnéztem mennyire egyszerű extendálni a rendszert új komponensekkel, és szerencsére elég egyszerű :)